### DISCUSSION TITLE
Modeling a nacelle explicitly using snappyHexMesh in conjunction with Actuator Line Model

### DISCUSSION THREAD

**Discussion by ssthakar** [turbinesFoam/turbinesFoam#363] [Q&A]
Hello, I am currently working on an implementation where a body with a rotor must accelerate and decelerate in water. This would mean I need to consider added mass effects for the nacelle. Is it possible to model the nacelle explicitly and employ a moving wall boundary condition on the nacelle while still using the standard ALM implementation of turbinesFOAM? 
Thank you

**Comment by petebachant** [comment]
Do you think you'd use a dynamic mesh to accelerate the nacelle?

**Comment by hcOnel** [comment]
That is an interesting case. Is your case a 6-degree-of-freedom case (nacelle will move inertially due to forces), or is it just going to move with a prescribed motion? If it is the latter, and the deformations are not large, utilizing a dynamic mesh solver might work. I wonder how would the ALM code work on a dynamic mesh in that case...
